src/main/java/sirius/biz/storage/layer2/BasicBlobStorageSpace.java

```diff
@@ -701,6 +730,24 @@ private void handleTenantMismatch(Object objectToModify, D newParent) {
                         .handle();
     }
 
+    /**
+     * Deletes the given directory.
+     *
+     * @param directory the directory to delete
+     */
+    public void deleteDirectory(D directory) {
```

Comment:
is this intentionally exposed as public. the official way is Directory.delete()

Reply:
Otherwise it is not possible to call deleteDirectory from SQLDirectory and MongoDirectory.

Reply:
k